internal/mempool/v1/mempool.go

```diff
@@ -722,6 +729,16 @@ func (txmp *TxMempool) insertTx(wtx *WrappedTx) {
 	txmp.txStore.SetTx(wtx)
 	txmp.priorityIndex.PushTx(wtx)
 
+	heightIdx := txmp.heightIndex.GetSortedIndex(func(curr *WrappedTx) bool {
```

Comment:
Can we add a test for the TTL removal from the mempool?

Reply:
This PR is in "Draft" mode -- it's not ready for review. Tests will come 😉

Reply:
Didn't catch the distinction! Thanks for clarifying